charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -13,6 +13,149 @@ Those components are needed to have the full experience on NROne kubernetes expl
+support different resources per component, therefore the value specified has been ignored.
+
+Please use ksm.resources, controlPlane.resources and kubelet.resources.
+
+{{- end }}
+
+{{- if .Values.tolerations }}
+
+###########
+# WARNING #
+###########
+
+You have specified "tolerations" into values, however the new version of the chart
+support different tolerations per component, therefore the value specified has been ignored.
+
+Please use ksm.tolerations, controlPlane.tolerations and kubelet.tolerations.
+
+{{- end }}
+
+{{- if ( or image.repository image.tag image.pullPolicy image.pullSecrets) }}
+
+###########
+# WARNING #
+###########
+
+You have specified into values options "image.*".
+The following values are no longer supported and are currently ignored.
+ - image.repository
+ - image.tag
+ - image.pullPolicy
+ - image.pullSecrets
+
+Notice that the 3.x version of the integration uses 3 different images.
+Please set:
+ - images.forwarder.* to configure the image in charge of sending data to newrelic backend
+ - images.agent.* to configure the image bundling the agent and onHost integration
+ - images.integration.* to configure the image in charge of scraping k8s data
+
+{{- end }}
+
+{{- if (or .Values.enableWindows .windowsOsList .windowsSecurityContext .windowsNodeSelector)}}
+
+###########
+# WARNING #
+###########
+
+Windows is currently supported by 2.x charts only and "windowsOsList" "windowsSecurityContext" "windowsNodeSelector"
+are no longer supported.
+
+{{- end }}
+
+{{- if ( or .Values.etcdTlsSecretName .etcdTlsSecretNamespace )}}
+
+###########
+# WARNING #
+###########
+
+Values "etcdTlsSecretName" and "etcdTlsSecretNamespace" are no longer supported, please specify them in the config
+file. Example:
+# - endpoints:
+#     - url: https://localhost:9979
+#       insecureSkipVerify: true
+#       auth:
+#         type: mTLS
+#         mtls:
+#           secretName: secret-name
+#           secretNamespace: secret-namespace
+
+{{- end }}
+
+{{- if .Values.logFile }}
+
+###########
+# WARNING #
+###########
+
+The "logFile" value is no longer supported, please specify the option "log_file" directly in "common.agentConfig".
+
+{{- end }}
+
+{{- if .Values.runAsUser }}
+
+###########
+# WARNING #
+###########
+
+The "runAsUser" value is no longer supported, please specify it with the value "securityContext.runAsUser".
+
+{{- end }}
+
+{{- if .Values.eventQueueDepth }}
+
+###########
+# WARNING #
+###########
+
+The "eventQueueDepth" value is no longer supported, please specify the option "event_queue_depth" directly in "common.agentConfig".
+
+{{- end }}
+
+{{- if .Values.apiServerSecurePort }}
+
+###########
+# WARNING #
+###########
+
+The "apiServerSecurePort" value is no longer supported, please specify it in the section "apiServer.autodiscover[].endpoints".
+
+{{- end }}
+
 {{- if and (include "newrelic.fargate" .) .Values.kubelet.affinity.nodeAffinity }}
```

Comment:
New chart is not compatible with fargate (yet), I'd add a clause to fail if it is specified
```
{{- if include "newrelic.fargate" . }}
Fargate is not supported for the v3 chart, please use v2 instead.
{{- fail "Fargate is not supported for the v3 chart, please use v2 instead" }}
{{- end }}

Reply:
why it is not? I would have said that this chart is as compatible as the old one.

Reply:
My thought was that were still injecting the "old" image as a sidecar, but it's true that the "rest" of the integration should work well 🤔